lua/nvim_lsp/tsserver.lua

```diff
@@ -2,7 +2,12 @@ local configs = require 'nvim_lsp/configs'
 local util = require 'nvim_lsp/util'
 
 local server_name = "tsserver"
-local bin_name = "typescript-language-server"
+local bin_name
+if vim.fn.has('win32') then
+  bin_name = "typescript-language-server.cmd"
+else
+  bin_name = "typescript-language-server"
+end
```

Comment:
You can write as follows:
```suggestion
local bin_name = "typescript-language-server"
if vim.fn.has('win32') then
  bin_name = bin_name..".cmd"
end
```